src/container/Login.jsx

```diff
@@ -0,0 +1,215 @@
+import React, { useEffect, useState} from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useNavigate } from 'react-router-dom'
+import { toast } from 'react-toastify'
+import { login, reset } from '../features/auth/authSlice'
+import { NavHeader } from '../components/index';
+import IconButton from "@mui/material/IconButton/IconButton";
+import Visibility from "@mui/icons-material/Visibility";
+import InputAdornment from "@mui/material/InputAdornment";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import Input from "@mui/material/Input";
+import { SpinnerCircular } from 'spinners-react';
+
+const Login = (props) => {
+
+  const [values, setValues] = React.useState({
+    email: "",
+    password: "",
+    showPasswordd: false,
+  });
+  
+  const [validation, setValidation] = React.useState({
+    email: "",
+    password: ""
+  });
+
+  const handleClickShowPassword = () => {
+    setValues({ ...values, showPasswordd: !values.showPasswordd });
+  };
+  
+  const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+  };
+  
+  const handleChange = (prop) => (event) => {
+    setValues({ ...values, [prop]: event.target.value });
+  };
```

Comment:
When there is an `error message` and user starts typing for retry, the `error message` should be removed from the UI. 
To achieve that you need to reset `error message` here

Reply:
on it